build.sbt

```diff
@@ -121,7 +121,6 @@ lazy val interopTests = Project(
   )
   .dependsOn(runtime)
   .enablePlugins(akka.grpc.ReflectiveCodeGen)
-  .enablePlugins(akka.grpc.NoPublish)
```

Comment:
Why do we want to publish the interop tests?

Reply:
They are used by the scripted tests. Might be more elegant to only `publishLocal` them, but can't really hurt either...